test-harness/specs/default_responses-AC-1.oas2.txt

```diff
@@ -0,0 +1,30 @@
+====test====
+When I send a request to an operation
+And this operation is defined
+And this operation has only a default response defined
+Then I should get back the default response
+====spec====
+swagger: "2.0"
+info:
+  description: "description"
+  title: "title"
+  version: "1.0"
+paths:
+  /todos:
+    get:
+      responses:
+        default:
+          description: default response
+          schema:
+            type: string
+            example: test
+====server====
+mock -p 4010
+====command====
+curl -i -X GET http://localhost:4010/todos
+====expect====
+HTTP/1.1 200 OK
+content-type: */*
```

Comment:
@philsturgeon I think this response is not correct

Reply:
oh damn, yeah, is this gavel just being too friendly? or is gavel not used here? 

Did we ever decide on how we'd implement the gavel-based expect, and the more strict expect?